app/code/Magento/Tax/Test/Mftf/ActionGroup/AdminDeleteTaxRateActionGroup.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminDeleteTaxRateActionGroup">
+        <annotations>
+            <description>Goes to the Admin Tax Rate grid page. Deletes the provided Tax Rate Code.</description>
+        </annotations>
+        <arguments>
+            <argument name="taxRateCode" type="string"/>
+        </arguments>
+
+        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters"/>
+        <fillField selector="{{AdminTaxRateGridSection.filterByTaxIdentifier}}" userInput="{{taxRateCode}}" stepKey="fillNameFilter"/>
+        <click selector="{{AdminTaxRateGridSection.search}}" stepKey="clickSearch"/>
+        <waitForPageLoad stepKey="waitForTaxRuleSearch"/>
+        <click selector="{{AdminTaxRateGridSection.nthRow('1')}}" stepKey="clickFirstRow"/>
+        <waitForPageLoad stepKey="waitForPageLoad"/>
+        <click selector="{{AdminTaxRateFormSection.deleteRate}}" stepKey="clickDeleteRate"/>
+        <click selector="{{AdminTaxRateFormSection.ok}}" stepKey="clickOk"/>
```

Comment:
Also, I would rather reuse the section `AdminMainActionsSection` and `AdminConfirmationModalSection` elements defined in Backend module than use a separate buttons defined in scope of each module.